pages/common/tlmgr-platform.md

```diff
@@ -0,0 +1,24 @@
+# tldr platform
+
+> Manage TeX Live platforms.
+> More information: <https://www.tug.org/texlive/tlmgr.html>.
+
+- List all available architectures in the package repository:
+
+`tlmgr platform list`
+
+- Add the executables for a specific architecture:
+
+`sudo tlmgr platform add {{platform}}`
```

Comment:
I feel that this sounds better
```suggestion
`sudo tlmgr platform add {{platform_name}}`
```

Reply:
Not sure this is necessary. Personally I'm not a fan of it.